installation/cmd/run.sh

```diff
@@ -13,7 +13,7 @@ if [[ $KYMA_RELEASE == *PR-* ]]; then
   if [ -z "$KYMA_TAG" ]; then echo "ERROR: Kyma artifacts for ${KYMA_RELEASE} not found."; exit 1; fi
   KYMA_SOURCE="eu.gcr.io/kyma-project/kyma-installer:${KYMA_TAG}"
 elif [[ $KYMA_RELEASE == master ]]; then
-  KYMA_SOURCE="latest-published"
+  KYMA_SOURCE="master"
```

Comment:
Why are we changing this? I do not think we want the installer to deploy the `master` version as it is not recognized as a stable one. Or is the `latest-published` related to the last commit on the master branch? I would need more insights on what is the goal here and how it will affect the process of installation (what will be installed on a cluster).

Reply:
It's just a renaming from `latest-published` to `master` as `master` is more obvious for the users.

Reply:
Just to clarify, this is the Kyma version that CLI is installing. So, it has nothing to do with `stable` binary for CLI. `latest-published` was installing the latest master from Kyma, so we renamed it to make it more obvious.